modules/afsocket/afsocket-source.c

```diff
@@ -397,12 +425,18 @@ afsocket_sd_process_connection(AFSocketSourceDriver *self, GSockAddr *client_add
 
 #endif
 
-  if (self->num_connections >= self->max_connections)
+  if (_connections_count_get(self) >= self->max_connections)
     {
+      gchar *source_name = cfg_tree_get_rule_name(&self->super.super.super.cfg->tree, ENC_SOURCE, self->super.super.super.expr_node);
+
       msg_error("Number of allowed concurrent connections reached, rejecting connection",
                 evt_tag_str("client", g_sockaddr_format(client_addr, buf, sizeof(buf), GSA_FULL)),
                 evt_tag_str("local", g_sockaddr_format(local_addr, buf2, sizeof(buf2), GSA_FULL)),
+                evt_tag_str("source name", source_name),
```

Comment:
please don't use spaces here. that confuses our log format. the proper solution is to use an underscore here.

Reply:
Fixed, thank you!